core/response.go

```diff
@@ -85,16 +86,10 @@ func (r *ProxyResponseWriter) GetProxyResponse() (events.APIGatewayProxyResponse
 		isBase64 = true
 	}
 
-	proxyHeaders := make(map[string]string)
-
-	for h := range r.headers {
-		proxyHeaders[h] = r.headers.Get(h)
-	}
-
 	return events.APIGatewayProxyResponse{
-		StatusCode:      r.status,
-		Headers:         proxyHeaders,
-		Body:            output,
-		IsBase64Encoded: isBase64,
+		StatusCode:        r.status,
+		MultiValueHeaders: textproto.MIMEHeader(r.headers),
```

Comment:
Why use the `MIMEHeader` from `textproto` instead of the `Header` object from `net/http`? They are both aliases of `map[string][]string`

Reply:
No strong reason for that. While browsing through Golang's source code I found that `Header` object from `net/http` basically reuses a lot of functions from `MIMEHeader` from `textproto`, and provides functionality I didn't need. (Ref: https://golang.org/src/net/http/header.go). Since I only needed a `map[string][]string`, I chose the lightweight one. 

Now I think more of that, I don't think it improves performance yet increases confusion in code. I've reverted to use `Header` from `net/http`.